src/Tensors.jl

```diff
@@ -7,7 +7,7 @@ using Statistics: mean
 using LinearAlgebra
 using StaticArrays
 # re-exports from LinearAlgebra
-export ⋅, ×, dot, diagm, tr, det, norm, eigvals, eigvecs, eigen
+export ⋅, ×, dot, diagm, tr, det, cof, norm, eigvals, eigvecs, eigen
```

Comment:
I don't think this come from LinearAlgebra so maybe list separately. Is `cof` a standard name for this operation btw?

Reply:
Oh, right. In the continuum mechanics books I read this operation is usually abbreviated with `cof` (Holzapfel, Wriggers). German Wikipedia also uses it.

https://de.wikipedia.org/wiki/Minor_(Lineare_Algebra)#Eigenschaften

But possible that this abbreviation is a German thing. If you want something different then we can also use it.

Reply:
I don't mind `cof`, just wondering if it was called that, or something else, in other software. I can't find anything from a quick serach though.

Reply:
My only worry here is that some standard library might include cofactors in the future. :D